File: 242.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()) return false;
        char[] char1 = s.toCharArray();
        char[] char2 = t.toCharArray();
        Arrays.sort(char1);
        Arrays.sort(char2);
        return Arrays.equals(char1, char2);
    }
}

File: 242.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        for i in range(len(s)):
            index = t.find(s[i])
            if index == -1:
                return False
            else:
                t = t[0:index] + t[index+1:]
        if not t:
            return True
        else:
            return False